plot_fluxnet.qmd

```diff
@@ -102,35 +111,33 @@ This is for all of the Arizona sites, which includes all the sites in Dave's lis
 
 ```{r}
 #| eval: false
-
-az_sites <- site_metadata %>% 
+az_sites <- site_metadata %>%
   filter(STATE == "AZ")
 
-downloaded_sites <- list.dirs("data") %>% 
+downloaded_sites <- dir_ls("data", glob = "*.zip") %>%
   str_split_i("_", 2)
 
-for(site in az_sites$SITE_ID){
-  if(!site %in% downloaded_sites){
-    print(paste0("Downloading data for ", site))
-    tryCatch({
-      amf_download_fluxnet(user_id = "kristinariemer",
-                           user_email = "kristinariemer@arizona.edu",
-                           site_id = site,
-                           data_product = "FLUXNET",
-                           data_variant = "FULLSET",
-                           data_policy = "CCBY4.0",
-                           agree_policy = TRUE,
-                           intended_use = "synthesis",
-                           intended_use_text = "creating pipeline for standardized figures",
-                           out_dir = "data/")
-      zip_path <- Sys.glob(file.path("data", paste0("AMF_", site, "*")))
-      unzipped_path <- tools::file_path_sans_ext(zip_path)
-      unzip(zip_path, exdir = unzipped_path)
-      }, error = function(e){cat("ERROR:",conditionMessage(e), "\n")})
-    } else {
-      print(paste0("Data has already been downloaded for ", site))
-    }
-}
```

Comment:
Didn't need a for loop here since `amf_download_fluxnet()` can take a vector of site IDs.  Not sure if there was another reason for doing this as a for loop.